packages/teleport/src/Audit/EventList/EventTypeCell.tsx

```diff
@@ -83,6 +83,10 @@ const EventIconMap: Record<EventCode, React.FC> = {
   [eventCodes.PORTFORWARD_FAILURE]: Icons.Info,
   [eventCodes.SUBSYSTEM]: Icons.Info,
   [eventCodes.SUBSYSTEM_FAILURE]: Icons.Info,
+  [eventCodes.RECOVERY_TOKEN_CREATED]: Icons.VpnKey,
```

Comment:
this should probably be `Icons.Info` b/c it is just a subset of user token, and reset password token also uses `Icons.Info`

Reply:
revised